content/30.ecosystem/60.network-faucets.md

```diff
@@ -11,6 +11,12 @@ To access the testnet funds (Sepolia) you can use one of the following third par
 Multi-Chain Faucet. You can use Chainstack Faucet to claim %%zk_testnet_name%% **0.05 ETH
 every 24 hours**. Chainstack API key is required.
 
+## %%zk_testnet_name%% faucet by thirdweb
+
+Connect your wallet to the [thirdweb Faucet](https://thirdweb.com/zksync-sepolia-testnet) and enjoy the free zkSync Sepolia! 
```

Comment:
Our brand name has been recently updated from zkSync to ZKsync!
```suggestion
Connect your wallet to the [thirdweb Faucet](https://thirdweb.com/zksync-sepolia-testnet) and enjoy the free ZKsync Sepolia! 
```

Reply:
@itsacoyote Corrected the spelling and removed the trailing white spaces!